charts/influxdb2/README.md

```diff
@@ -1,12 +1,10 @@
 # InfluxDB
 
-**Warning**: This chart, as well as the software it deploys, is in a BETA phase.
+**Warning**: This InfluxDB Helm chart and the software it deploys are in a beta phase.
```

Comment:
Are all of our Helm charts in beta?

Reply:
No, just this one

Reply:
It looks like we say "InfluxData Helm charts are in beta and subject to change. " on the main readme....

Reply:
Lets remove the global

Reply:
thanks, @rawkode - removed from main README